### ISSUE TITLE
Admin login error in edge environment

### ISSUE THREAD

**Issue by rawaa123** [ude-soco/CourseMapper-webserver#1180]
@ralf-berger Please, we tried to access the admin account in Edge, but we were unable to log in. It gives the error message: 'User not found.' as the admin account is not created.

**Comment by rawaa123** [comment]
@ralf-berger The admin account is created automatically when we launch the server for the first time. However, if the data is deleted and we continue collecting data without relaunching the server, the admin account will also be deleted and not re-created. Now, the admin account is for some reason is omitted and we are not able to access it.

**Comment by rawaa123** [comment]
@ralf-berger The statements: '3 Roles present, skipping initialization' and '49 Users present, skipping initialization' will check the roles and the number of users currently present in the system, confirming that there are 3 roles but without verifying if the admin account is created.

The web server appears to be functioning correctly and executes as expected.

Moreover, these updates are from 11 October 2024, but I believe we launched the edge branch in March, correct? This means the admin account must have been created at that time and should not have been deleted.

If, for any reason, we deleted the old data in MongoDB, the admin account would also have been deleted without triggering an error, and the system would continue running smoothly. Any suggestions?
